plugins/updater/src/commands.rs

```diff
@@ -40,6 +40,8 @@ pub(crate) struct Metadata {
 struct DownloadedBytes(pub Vec<u8>);
 impl Resource for DownloadedBytes {}
 
+// TODO: Align this with the result of `updater.check` to Result<Option<Metadata>>
+// and remove `available` instead of handling this in the js side
```

Comment:
I don't know if it's considered breaking if we change this from returning `Result<Metadata>` to `Result<Option<Metadata>>`, if not then I would like to do it before v3

Reply:
We can change it, if you want since we now require users to sync their rust and js versions

Reply:
That's nice, I'll do it later